### PR TITLE
fix network split between seed nodes

### PR DESCRIPTION
**Pull request by laizy**

### CODE REVIEW

File: p2pserver/p2pserver.go [hunk 1]
```diff
@@ -305,25 +304,37 @@ func (this *P2PServer) connectSeeds() {
...
-	if len(pList) > 0 {
+
+	if len(seedConnList) > 0 {
 		rand.Seed(time.Now().UnixNano())
-		index := rand.Intn(len(pList))
-		this.reqNbrList(pList[index])
+		index := rand.Intn(len(seedConnList))
+		this.reqNbrList(seedConnList[index])
+		if len(seedDisconn) > 0 {
+			index := rand.Intn(len(seedDisconn))
```
Comment (Honglei-Cong) [inline p2pserver/p2pserver.go:340]: only enable this for consensus node?

Comment (Honglei-Cong) [reply p2pserver/p2pserver.go:340]: or add another 'fullConnectionFlag', enable the flag if it is consensus node?

Comment (Honglei-Cong) [reply p2pserver/p2pserver.go:340]: or add another 'fullConnectionFlag', enable the flag if it is consensus node?

Comment (laizy) [reply p2pserver/p2pserver.go:340]: implemented seed node check

**Review by Honglei-Cong — APPROVED** [review]

**Comment by laizy** [comment]
close and reopen to rerun jenkins

**Pull request merged**